File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/Adjustment.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class Adjustment {
    @JsonProperty("AdjustedByUserName")
    public String adjustedByUserName;
    @JsonProperty("AdjustmentDate")
    public String adjustmentDate;
    @JsonProperty("AdjustmentNote")
    public String adjustmentNote;
    @JsonProperty("AdjustmentQuantity")
    public Double adjustmentQuantity;
    @JsonProperty("AdjustmentReasonName")
    public String adjustmentReasonName;
    @JsonProperty("AdjustmentUnitOfMeasureAbbreviation")
    public String adjustmentUnitOfMeasureAbbreviation;
    @JsonProperty("AdjustmentUnitOfMeasureName")
    public String adjustmentUnitOfMeasureName;
    @JsonProperty("ItemCategoryName")
    public String itemCategoryName;
    @JsonProperty("ItemName")
    public String itemName;
    @JsonProperty("PackageId")
    public Integer packageId;
    @JsonProperty("PackageLabTestResultExpirationDateTime")
    public String packageLabTestResultExpirationDateTime;
    @JsonProperty("PackageLabel")
    public String packageLabel;
    @JsonProperty("PackagedDate")
    public String packagedDate;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/Adjustment.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class Adjustment(
    @JsonProperty("AdjustedByUserName")
    val adjustedByUserName: String? = null,
    @JsonProperty("AdjustmentDate")
    val adjustmentDate: String? = null,
    @JsonProperty("AdjustmentNote")
    val adjustmentNote: String? = null,
    @JsonProperty("AdjustmentQuantity")
    val adjustmentQuantity: Double? = null,
    @JsonProperty("AdjustmentReasonName")
    val adjustmentReasonName: String? = null,
    @JsonProperty("AdjustmentUnitOfMeasureAbbreviation")
    val adjustmentUnitOfMeasureAbbreviation: String? = null,
    @JsonProperty("AdjustmentUnitOfMeasureName")
    val adjustmentUnitOfMeasureName: String? = null,
    @JsonProperty("ItemCategoryName")
    val itemCategoryName: String? = null,
    @JsonProperty("ItemName")
    val itemName: String? = null,
    @JsonProperty("PackageId")
    val packageId: Int? = null,
    @JsonProperty("PackageLabTestResultExpirationDateTime")
    val packageLabTestResultExpirationDateTime: String? = null,
    @JsonProperty("PackageLabel")
    val packageLabel: String? = null,
    @JsonProperty("PackagedDate")
    val packagedDate: String? = null
)
